v2/samples/synapse/v1beta/v1beta20210601_synapsebigdatapool.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: synapse.azure.com/v1beta20210601
```

Comment:
Nice! you'd also need to run the tests on these samples to generate a recording. You can follow the [documentation](https://azure.github.io/azure-service-operator/contributing/add-a-new-code-generated-resource/#run-test-for-added-sample-and-commit-the-recording) to do so.